sdk/identity/azure-identity/src/main/java/com/azure/identity/DefaultAzureCredential.java

```diff
@@ -35,5 +40,18 @@
      */
     DefaultAzureCredential(ArrayDeque<TokenCredential> tokenCredentials) {
         super(tokenCredentials);
+        this.tokenCredentials = new ArrayList<TokenCredential>(tokenCredentials.size());
+        this.tokenCredentials.addAll(tokenCredentials);
+    }
+
+    /**
+     * Get the list of credentials sequentially used by {@link DefaultAzureCredential} to attempt authentication.
+     * The credentials in the returned list and their order may change in future versions of Identity.
```

Comment:
javadoc : `Get a list of read-only credentials sequentially `  ? Since it is `unmodifiableList`

Reply:
not unomodifiable anymore.